src/components/EnterpriseApp/index.jsx

```diff
@@ -141,7 +143,7 @@ class EnterpriseApp extends React.Component {
                   tabIndex="-1"
                   ref={this.contentWrapperRef}
                   style={{
-                    paddingLeft: matchesMediaQ ? sidebarWidth : defaultContentPadding,
+                    [isLocaleRTL ? 'paddingRight' : 'paddingLeft']: matchesMediaQ ? sidebarWidth : defaultContentPadding,
```

Comment:
[suggestion] Rather than relying on `getLocale` and `isRtl` here, I would recommend a pure CSS solution with `paddingInlineStart` here instead, the primary reasons being:
1. It's a simpler implementation.
2. Can respond to manual changes of the `dir="ltr"` attribute on the `<html>` element, whereas this solution would not. For example, if you manually change the `<html>` element's `dir="ltr"` attribute in DevTools to `dir="rtl"`, the padding is not properly updated with the current approach.

The `padding-inline-start` CSS property ([docs](https://developer.mozilla.org/en-US/docs/Web/CSS/padding-inline-start)) takes into account LTR vs. RTL directionality by default.

Example screenshot, after manually changing to `dir="rtl"` in DevTools and relying on `paddingInlineStart`:

![image](https://github.com/openedx/frontend-app-admin-portal/assets/2828721/4f2bddb6-38b4-4c5c-bc33-bca81041f530)